### DISCUSSION TITLE
Multi-label Images

### DISCUSSION THREAD

**Discussion by clindhorst2** [ultralytics/ultralytics#5198] [Q&A]
I know in YOLO v5 you could use multi-class labels but, then you would get multiple bounding boxes.  Has it been changed so that multiple labels use the same bounding box? Do you still add the multiple labels the same way?

Ex:
3 0.34 0.24  0.55 0.49
6 0.34 0.24  0.55 0.49

**Comment by glenn-jocher** [comment]
@clindhorst2 hello! In YOLOv8, the handling of multi-label annotations remains consistent with the approach you're familiar with from YOLOv5. If you have instances where an object should have multiple class labels, you can indeed annotate them with the same bounding box coordinates, as you've shown in your example. This will result in the model predicting multiple classes for the same object location during inference. The annotation format for multi-labels hasn't changed, so you can continue to add them in the same way. If you need more details on annotation formats and best practices, please refer to our documentation. Happy training! 😊🚀
